api.swagger.yaml

```diff
@@ -1845,6 +1845,51 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/manuals/{manual_page_id}/pages':
+    get:
+      description: Get all pages in a manual or manual page.
+      tags:
+        - manuals
+      parameters:
+        - name: latest
+          description: Whether to return the latest unpublished revision of each page.
+          in: query
+          schema:
+            type: boolean
+            default: false
+        - name: cursor
+          description: The paging cursor.
+          in: query
+          schema:
+            type: string
+            format: uuid
```

Comment:
Cursor should be opaque (it can be _implemented_ as a UUID but the API consumer should not rely on this)

Reply:
Fixed.